dts/bindings/rng/zephyr,psa-crypto-rng.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: PSA Crypto RNG/Entropy
```

Comment:
This and the commit message need to be more descriptive. Why do you need a zephyr-specific binding? What platforms is this applicable to? Etc.

Reply:
Ack, concerning the zephyr-specific binding, should it the prefix be zephyr or arm ?

Reply:
This looks like it's a binding for a zephyr-specific software construct that happens to run on Arm hardware. Is that right? If so, I think `zephyr,`.